src/e2e-tests/page-objects/autelis.page.js

```diff
@@ -7,200 +7,182 @@ let counter = 0;
-    get switchMessageSolar() {return $('//div[text()[starts-with(., "Solar")]]')}
-
-    get solarOnButton() {return $('//div[text()[starts-with(., "Solar")]]/parent::*//button[text()="On"]')}
-    get solarOffButton() {return $('//div[text()[starts-with(., "Solar")]]/parent::*//button[text()="Off"]')}
-    get cleanerOnButton() {return $('//div[text()="Cleaner"]/parent::*//button[text()="On"]')}
-    get cleanerOffButton() {return $('//div[text()="Cleaner"]/parent::*//button[text()="Off"]')}
-
-    get poolHeatOnButton() {return $('//div[text()="Pool Heat"]/parent::*//button[text()="On"]')}
-    get poolHeatOffButton() {return $('//div[text()="Pool Heat"]/parent::*//button[text()="Off"]')}
-    get poolLightOnButton() {return $('//div[text()="Pool Light"]/parent::*//button[text()="On"]')}
-    get poolLightOffButton() {return $('//div[text()="Pool Light"]/parent::*//button[text()="Off"]')}
-    get waterfallOnButton() {return $('//div[text()="Waterfall"]/parent::*//button[text()="On"]')}
-    get waterfallOffButton() {return $('//div[text()="Waterfall"]/parent::*//button[text()="Off"]')}
-
-    get spaHeatOnButton() {return $('//div[text()="Spa Heat"]/parent::*//button[text()="On"]')}
-    get spaHeatOffButton() {return $('//div[text()="Spa Heat"]/parent::*//button[text()="Off"]')}
-    get jetsOnButton() {return $('//div[text()="Jets"]/parent::*//button[text()="On"]')}
-    get jetsOffButton() {return $('//div[text()="Jets"]/parent::*//button[text()="Off"]')}
-    get spaLightOnButton() {return $('//div[text()="Spa Light"]/parent::*//button[text()="On"]')}
-    get spaLightOffButton() {return $('//div[text()="Spa Light"]/parent::*//button[text()="Off"]')}
-    get blowerOnButton() {return $('//div[text()="Blower"]/parent::*//button[text()="On"]')}
-    get blowerOffButton() {return $('//div[text()="Blower"]/parent::*//button[text()="Off"]')}
-
-    // 0 - lower; 1 - higher
-    get poolHeatAdjButtons() {return $$('//div[text()="Pool Heat"]/parent::*//div[@class="float-right"]/button')}
-    get poolHeatCounter() {return $('//div[text()="Pool Heat"]/parent::*//div[@class="float-right"]/input')}
-    // 0 - lower; 1 - higher
-    get spaHeatAdjButtons() {return $$('//div[text()="Spa Heat"]/parent::*//div[@class="float-right"]/button')}
-    get spaHeatCounter() {return $('//div[text()="Spa Heat"]/parent::*//div[@class="float-right"]/input')}
+    get switchMessagePool() {return $('//*[contains(@class, "AutelisTab_mainSwitch")]/parent::*//div[text()[starts-with(., "Pool")]]')}
+    get switchMessageSpa() {return $('//*[contains(@class, "AutelisTab_mainSwitch")]/parent::*//div[text()[starts-with(., "Spa")]]')}
+    get switchMessageSolar() {return $('*//div[text()[starts-with(., "Solar")]]')}
+
+    // 0 - Solar; 1 - Cleaner; 2 - Pool Heat; 3 - Pool Light; 4 - Waterfall; 5 - Spa Heat;
+    // 6 - Jets; 7 - Spa Light; 8 - Blower
+    get onButton() {return $$('//*[contains(@class, "AutelisTab_toggle")]//ion-label[text()="On"]')}
```

Comment:
Please keep on and off buttons as they were, section based. 
Here is an example of an xPath that works for `cleanerOffButton`:
`//div[text()="Cleaner"]/parent::*//ion-segment-button[.="On"]`

Please note the difference between `text()` and `.` in xPath

Reply:
Changed.

Reply:
Changed.